internal/config/colorscheme.go

```diff
@@ -81,22 +85,42 @@ func LoadColorscheme(colorschemeName string) error {
 // Colorschemes are made up of color-link statements linking a color group to a list of colors
 // For example, color-link keyword (blue,red) makes all keywords have a blue foreground and
 // red background
-func ParseColorscheme(text string) (map[string]tcell.Style, error) {
+func ParseColorscheme(name string, text string) (map[string]tcell.Style, error) {
 	var err error
-	parser := regexp.MustCompile(`color-link\s+(\S*)\s+"(.*)"`)
+	colorParser := regexp.MustCompile(`color-link\s+(\S*)\s+"(.*)"`)
+	includeParser := regexp.MustCompile(`include\s+"(.*)"`)
+	ParsedColorschemes = append(ParsedColorschemes, name)
 
 	lines := strings.Split(text, "\n")
 
 	c := make(map[string]tcell.Style)
 
+lineLoop:
 	for _, line := range lines {
 		if strings.TrimSpace(line) == "" ||
 			strings.TrimSpace(line)[0] == '#' {
 			// Ignore this line
 			continue
 		}
 
-		matches := parser.FindSubmatch([]byte(line))
+		matches := includeParser.FindSubmatch([]byte(line))
+		if len(matches) == 2 {
+			include := string(matches[1])
+			for _, name := range ParsedColorschemes {
+				// check for circular includes...
+				if name == include {
+					// ...and prevent them
+					continue lineLoop
+				}
+			}
+			err = LoadColorscheme(include)
+			if err == nil {
+				c = Colorscheme
+			}
+			continue
```

Comment:
What if `err != nil`?

Reply:
The best is to stop parsing here and inform the user about the broken include chain.